app/form_models/vacancy_algolia_search_form.rb

```diff
@@ -5,7 +5,7 @@ class VacancyAlgoliaSearchForm
               :location, :location_category, :radius,
               :job_roles, :phases, :working_patterns,
               :jobs_sort, :page,
-              :job_role_options, :school_phase_options, :working_pattern_options,
+              :job_role_options, :education_phase_options, :working_pattern_options,
```

Comment:
Even though the frontend says `Education phase`, I'd prefer the attribute to be called `school_phase_options`, since we're referring to the `phase` on the `School`

Reply:
A side-effect of this would be that, given the iteration over an array in pages/home/_filters.html.haml, the en.yml would also now want to label the words 'Education phase' as '**school_phase**'.

I am now clocking off for the rest of the sprint and don't mind either way, and don't mind the third way of calling it 'phase' everywhere, which was my original choice.